src/main.c

```diff
@@ -1319,6 +1325,11 @@ main(int argc, char *argv[])
         np2_stderr_log = 0;
     }
 
+    /* use syslog even if running in foreground */
```

Comment:
Inaccurate now, it is not only about the `-F` parameter.

Reply:
Fixing ...